configure.ac

```diff
@@ -95,7 +114,9 @@ AC_PROG_INSTALL
 AC_PROG_LN_S
 AC_PROG_MAKE_SET
 AM_PROG_AR
-AC_PROG_LIBTOOL
+
+#AC_PROG_LIBTOOL - obsolete
+LT_INIT()
 
 
 # Checks for libraries.
```

Comment:
I was considering this as well and I guess it could be made optional - just each such change in the build system(s) require quite a bit of time (multiple systems to tests...).

The simplest way to build ADFlib not having the Check framework in the system is to build with CMake - it will pull Check directly from its repo, if it is not found in the system.

Reply:
@kyz tell me if you want more access to this repo

Reply:
Hi @lclevy I think I already have access, and I'd be happy to work on a feature like this, but I'd like to see this PR merged first, currently I'm just giving my opinions in a review